support-frontend/assets/helpers/redux/commonState/selectors.ts

```diff
@@ -55,3 +55,13 @@ export function isUserInAbVariant(abTestName: string, variantName: string) {
 		return participations[abTestName] === variantName;
 	};
 }
+
+export function isUserInAbVariants(abTestNames: string[], variantName: string) {
+	return function getAbTestStatus(state: ContributionsState): boolean {
+		const participations = state.common.abParticipations;
+		const foundAbTests = abTestNames.filter(
+			(abTestName) => participations[abTestName] === variantName,
+		);
+		return foundAbTests.length > 0;
```

Comment:
```suggestion
		const matchingAbTest = abTestNames.find(
			(abTestName) => participations[abTestName] === variantName,
		);
		return !!matchingAbTest
```
It doesn't really matter with such tiny arrays, but if you have a condition where 'as long as at least one of these array items matches the condition, go ahead', `find` is more efficient because it will return as soon as it finds the first match rather than continuing to loop through the whole array.

I'd also suggest tweaking the function name from `isUserInAbVariants` to something that more clearly communicates that we're checking if the user is in `variantName` for _any one of_ the passed tests; I feel like the existing name could also be taken for a function that's checking if the user is in _all_ of the passed tests.

Reply:
Done